scss/style.css

```diff
@@ -0,0 +1,650 @@
+* {
+  margin: 0;
+  padding: 0;
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+.visually-hidden {
```

Comment:
Överväg att komprimera filen https://youtu.be/uvlytWYV6Jk

Reply:
Tack! Fixat!